### ISSUE TITLE
Optional to generate scss

### ISSUE THREAD

**Issue by horseyhorsey** [BionicFramework/BionicCLI#1]
Is it possible for a flag to specify whether you want to generate .scss for component & pages?

Currently if you don't want the .scss for a component then the file has to be deleted and entry removed from App.scss. I find it's putting more files and work in when you might not need them.

**Comment by bmsantos** [comment]
@horseyhorsey, currently it is not possible but I will mark this ticket as a future request.
Will try to add it in the next couple of days.

**Comment by bmsantos** [comment]
@horseyhorsey, this option is now available in Bionic 1.0.26.

You can use ```-n``` | ```--no-styles```:
```bash
bionic generate component MyComponent -n
bionic generate layout MyLayout --no-styles
bionic generate page MyPage -n
```

**Comment by horseyhorsey** [comment]
Tested and thanks!
